src/controllers/MainController.js

```diff
@@ -0,0 +1,31 @@
+import InputView from '../views/InputView.js';
+import OutputView from '../views/OutputView.js';
+import { InputValidator } from '../models/validations/InputValidator.js';
+import { isEmptyInput } from '../models/validations/isEmptyInput.js';
+import { updateSeparators } from '../utils/updateSeparators.js';
+import { parsingWithSeparators, parsingNumberPart } from '../utils/parser.js';
+import { Calculator } from '../models/Calculator.js';
+
+export class MainController {
+	async run() {
+		try {
+			const rawInput = await InputView.readInput();
+			if (isEmptyInput(rawInput)) return OutputView.printResult(0);
+			InputValidator.validateStrings(rawInput);
+			const separators = updateSeparators(rawInput, [':', ',']);
+			const numbers = parseInput(rawInput);
+			InputValidator.validateNumbers(numbers);
+			const result = Calculator.calculate(numbers, separators);
+			OutputView.printResult(result);
```

Comment:
```suggestion
			const rawInput = await InputView.readInput();
			
			if (isEmptyInput(rawInput)) return OutputView.printResult(0);
			
			InputValidator.validateStrings(rawInput);
			
			const separators = updateSeparators(rawInput, [':', ',']);
			const numbers = parseInput(rawInput);
			
			InputValidator.validateNumbers(numbers);
			
			const result = Calculator.calculate(numbers, separators);
			
			OutputView.printResult(result);
```

줄바꿈을 하지 않으신 이유가 있으실까요?

어느정도 문맥에 따라서 줄바꿈이 들어가면 가독성이 더 좋아질 거 같아요

Reply:
~~졸았습니다2~~

좋은 피드백 감사합니다!